api/v1/controllers/events.js

```diff
@@ -74,4 +76,39 @@ module.exports = {
     doPost(req, res, next, helper);
   },
 
+  /**
+   * POST /events/bulk
+   *
+   * Upserts multiple events. Returns "OK" without waiting for the creates to
+   * happen.
+   *
+   * @param {IncomingMessage} req - The request object
+   * @param {ServerResponse} res - The response object
+   * @param {Function} next - The next middleware function in the stack
+   * @returns {Promise} - A promise that resolves to the response object,
+   * indicating merely that the bulk create request has been received.
+   */
+  bulkPostEvent(req, res, next) {
+    const resultObj = { reqStartTime: req.timestamp };
+    const value = req.swagger.params.queryBody.value;
+    const body = { status: 'OK' };
+
+    /**
+     * Performs bulk upsert through worker, cache, or db model.
+     * Works regardless of whether user if provided or not.
+     *
+     * @param {Object} user Sequelize result. Optional
+     * @returns {Promise} a promise that resolves to the response object
+     * with status and body
+     */
+    function bulkPost(user) {
+      helper.model.bulkCreate(value, user);
```

Comment:
no way to know if this succeeds or fails?

Reply:
@iamigo I was following the same pattern as here: https://github.com/salesforce/refocus/blob/master/api/v1/controllers/samples.js#L448, except for in this PR we won't be using worker dynos. What would the action be if this succeeds/fails, as the response has already been sent to the client at this stage.

Reply:
The response says "ok we got your request" but it can't say "yes, we saved these events to the db" -- what if there are errors saving the records? That's why for bulk operations we give users the ability to get job status with array of errors if any